src/components/common/SimpleTable/index.tsx

```diff
@@ -85,11 +101,6 @@ const Wrapper = styled.table<{ $numColumns?: number }>`
       display: flex;
       align-items: center;
       justify-content: flex-end;
-
-      &:first-of-type {
-        padding-left: 1rem;
-        justify-content: flex-start;
-      }
```

Comment:
First column no longer aligns left then:
![screenshot_2021-02-02_13-05-22](https://user-images.githubusercontent.com/43217/106662514-52c2f280-6557-11eb-89f5-db73b6dcbbb9.png)
Storybook https://pr102--gpui.review.gnosisdev.com/storybook/?path=/docs/common-simpletable--custom-table-style
Trade app https://pr102--gpui.review.gnosisdev.com/trade.html

Is that ok?

Reply:
Good call, forgot that simpleTable is used by this component in the classic interface. Will fix.

Reply:
Should be fixed. As for fine tuning the classic UI table, I'd suggest we address it in a separate PR.